UI/RemoteISOScreen.cpp

```diff
@@ -598,8 +600,8 @@ RemoteISOBrowseScreen::RemoteISOBrowseScreen(const std::vector<std::string> &gam
 void RemoteISOBrowseScreen::CreateViews() {
 	bool vertical = UseVerticalLayout();
 
-	I18NCategory *mm = GetI18NCategory("MainMenu");
 	I18NCategory *di = GetI18NCategory("Dialog");
+	I18NCategory *n = GetI18NCategory("Networking");
```

Comment:
Well, these are the existing ones:

https://github.com/hrydgard/ppsspp/blob/ec7e6342151e9303eeafcf54ba79cfb4c8f757e1/UI/GameSettingsScreen.cpp#L613-L626
https://github.com/hrydgard/ppsspp/blob/d30116b8c3f508a1696d7da39e5ab5dd874ad524/Core/HLE/proAdhoc.cpp#L1461

And then here for the other type:
https://github.com/hrydgard/ppsspp/blob/c35461b75cceadcdea1709a9866600990227276f/UI/RemoteISOScreen.cpp#L678-L687

In settings, the heading is "Networking" so I'd vote on changing the remote iso feature - maybe even `[RemoteISO]` and `rs`?

-[Unknown]